quilt/src/main/resources/summonerscrolls.mixins.json

```diff
@@ -1,7 +1,8 @@
 {
   "required": true,
-  "package": "net.examplemod.mixin.forge",
+  "package": "com.github.teamfusion.summonerscrolls.mixin.fabric",
```

Comment:
I've never modded Quilt before, so just want to make sure this should be fabric and not Quilt. I know they're built on the same base, so I could still see this being correct.

Reply:
yeah it should be fine, irrc the template comes like this